specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,959 @@
+        "operationId": "Features_GetItems",
+        "x-ms-examples": {
+          "GetFeatures": {
+            "$ref": "./examples/features/Features_GetItems.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/LimitFeatures"
+          },
+          {
+            "$ref": "#/parameters/Bbox"
+          },
+          {
+            "$ref": "#/parameters/Filter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/ExtendedGeoJsonFeatureCollection"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      },
+      "post": {
+        "description": "The Post Feature API creates a feature with the provided attributes in the given collection.\n\nThe Features API is a part of [Creator](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps), which makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset/create). Datasets consist of multiple feature collections, which are a collection of features of a similar type based on a common schema referred to as ontology. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).",
```

Comment:
Request Body for Features Create should have a row for Properties. 
Similarly, Features definition should have a row for Properties.

Reply:
I'm a bit confused here, which lines are you referring to exactly?

Reply:
I was referring to this section:
![image](https://user-images.githubusercontent.com/35284488/210878257-9d138147-b7ef-4de4-8c86-d6436afe9ce2.png)
Not sure where it is located. 
For Features, I was referring to this section:
![image](https://user-images.githubusercontent.com/35284488/210878460-cf86a3b5-5332-40ff-8f6f-e05cb526d57d.png)

Same applies to other comments

Reply:
So after doing some digging, it appears that the request body definition I defined in [#parameters/FeatureBody](https://github.com/Azure/azure-rest-api-specs/pull/19539/files#diff-bebed0270ce3522f925f7208ea9dce74c2e5d8faaecc10fc0a59c12a9e9dc9b9R116) isn't being interpreted correctly. I'm still investigating what the error is, so pending updates inbound.

Reply:
Found the issue and now properties is rendered properly. Thanks for catching that!